docs/developer-docs/integrations/exchange-rate/exchange-rate-canister.md

```diff
@@ -0,0 +1,134 @@
+# Exchange rate canister
+
+## Overview
+
+The exchange rate canister, referred to as the XRC, is a canister that runs on the **uzr34 system subnet**. The XRC uses [HTTPS outcalls](https://internetcomputer.org/https-outcalls/) to fetch data from major cryptocurrency exchanges by using the exchange's public API to retrieve real time or historical cryptocurrency pricing information. The XRC also queries the public APIs for foreign exchange data providers around the world periodically in order to get forex rates. 
+
+The XRC canister can be integrated with decentralized exchanges (DEXs) to provide functionality such as comparing exchange rates against market rates to determine the value of assets that are held in a canister smart contract. For example, the cycle minting canister of the NNS uses the XRC to obtain the current ICP/XDR exchange rates, which it requires for the conversion of ICP to cycles.
```

Comment:
"can be integrated with" could be misunderstood. Dapps like DEXs can integrate the XRC (to make use of its service) but the XRC is unaware of the applications it serves.